staking/integration-tests/tests/pool_authority.rs

```diff
@@ -128,6 +129,76 @@ fn test_pool_authority() {
     );
 
 
+    let create_position_tx = Transaction::new_signed_with_payer(
+        &[create_position_ix],
+        Some(&payer.pubkey()),
+        &[&payer, &pool_authority],
+        svm.latest_blockhash(),
+    );
+
+    assert_anchor_program_error!(
+        svm.send_transaction(create_position_tx),
+        ErrorCode::UnexpectedTargetAccount,
+        0
+    );
```

Comment:
this test is duplicated below

Reply:
the name was wrong, it is actually testing close position. Fixed.